### ISSUE TITLE
Feature request: a11y - show population pourcentage in Accessibility Addon

### ISSUE THREAD

**Issue by Aarbel** [storybookjs/storybook#15866]
Color-blinded Population pourcentage should be shown in Accessibility plugin.

Why ? To see if we test accessibility with a big pourcentage of color-blinded population, or people with low vision. 

For example:
- maybe people with blurred vision on a device are much more than what we expect (idk, maybe 2% of web users), so maybe focusing more on making texts / buttons bigger should bring a better accessibility.
- maybe Tritanomaly just concerns 0,0001 % of we user, so maybe not a big thing to constraint too much the UI with this accessibility criteria (but surely make it readable for them)

![image](https://user-images.githubusercontent.com/25119847/129775571-f1d8ceef-27fa-4595-ae9b-69313626fcd5.png)

**Comment by Aarbel** [comment]
Cf https://en.wikipedia.org/wiki/Color_blindness

- Normal Vision - 92%
- Deuteranomaly - 2%
- Protanomaly - 0,66%
- Protanopia - 0,59%
- Deuteranopia - 0,56%
- Tritanopia - 0,016%
- Tritanomaly - 0,01%
- Achromatopsia - 0,0001%

**Comment by Aarbel** [comment]
Can't find metrics about blurred vision, if someone can help me

```
uncorrected refractive errors
cataract
age-related macular degeneration
glaucoma
diabetic retinopathy
corneal opacity
trachoma
```

**Comment by Aarbel** [comment]
Cf https://github.com/storybookjs/storybook/blob/07c8974649e0fa95ce1fe8e1b8a0e553754bbdb0/addons/a11y/src/components/VisionSimulator.tsx

**Comment by Aarbel** [comment]
@maya maybe you can help ?

**Comment by shilman** [comment]
cc @MichaelArestad

**Comment by jsomsanith** [comment]
Hey everyone,
What about this mockup ?
<img width="217" alt="image" src="https://user-images.githubusercontent.com/2874505/163717917-4da1c27d-06f1-45e7-86ae-317cffa8c85d.png">

Agree that this is not some critical info that changes everytime, and an update maintenance each year doesn't cost anything instead of getting it from api

**Comment by MichaelArestad** [comment]
@jsomsanith I dig this mockup! I might change it to `2,7% of users` (no "the")

**Comment by jsomsanith** [comment]
👍 I can ermove the `the`
If the visual is ok, I can work on a PR

**Comment by Aarbel** [comment]
@jsomsanith for blurred vision (maily myopa):

22.9%

https://www.aaojournal.org/article/s0161-6420(16)00025-7/pdf#:~:text=We%20estimated%201406%20million%20people,e%206.3%25%5D)%20in%202000.

**Comment by Aarbel** [comment]
🙌

**Comment by shilman** [comment]
Ta-da!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.5.0-rc.0 containing PR #18003 that references this issue. Upgrade today to the `@next` NPM tag to try it out!

```
npx sb upgrade --prerelease
```
